File: Iniciante/1097 - Sequencia IJ 3/JavaScript/1097.js
function main() {
  let I = 1;
  let JInicial = 7;

  while (I <= 9) {
    let contador = 1;
    let J = JInicial;
    while (contador <= 3) {
      console.log(`I=${I} J=${J}`);
      J--;
      contador++;
    }
    I += 2;
    JInicial += 2;
    contador = 1;
  }
}

main();

File: Iniciante/1097 - Sequencia IJ 3/Python/1097.py
def main():
    I = 1
    J_inicial = 7
    while I <= 9:
        contador = 1
        J = J_inicial
        while contador <= 3:
            print(f"I={I} J={J}")
            J -= 1
            contador += 1
        I += 2
        J_inicial += 2
        contador = 1


main()
